core/src/main/java/net/adoptopenjdk/icedteaweb/jvm/JvmUtils.java

```diff
@@ -276,7 +287,10 @@ public static boolean isValidSecureProperty(final String argument) {
                 "sun.java2d.win.uiScaleX",
                 "sun.java2d.win.uiScaleY",
                 "sun.java2d.uiScale",
-                "prism.allowhidpi" // for JavaFX
+                "prism.allowhidpi", // for JavaFX
+                "sun.net.client.defaultConnectTimeout",
+                "sun.net.client.defaultReadTimeout",
+                "java.locale.providers"
```

Comment:
Can you remove the `java.locale.providers`.
See OWS-465 for the security relevant restrictions.

Best to put this in a separate PR